diagnostic.md

```diff
@@ -5,74 +5,80 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+to handle and hold the information so that it can relay it
+to the front end to help interact with the client request.  example would be
+client wants to read a data the backend would gather and send it to front end to
+view
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+router
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+the controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Views are merely puppets reading what the controller gives them.
```

Comment:
Hah I like this answer but the reason we don't use rails views is because we manage our view states on the front-end.